app/presenters/hyrax/collapsable_section_presenter.rb

```diff
@@ -26,7 +26,7 @@ def button_tag
 		  data: { toggle: 'collapse' },
 		  href: "##{id}",
 		  onclick: "toggleCollapse(this)",
-		  'aria-expanded' => open,
+		  aria: { expanded: true },
 		  'aria-controls' => id) do
```

Comment:
Since you went in and changed the aria definition syntax, can you move the aria-controls into here for consistency?